src/core/Notification/Notification.md

```diff
@@ -6,40 +6,42 @@ import {
 } from 'suomifi-ui-components';
 
 <>
-  <Notification closeText="Close Et turpis eget tempor. Et turpis eget tempor. Et turpis eget tempor.">
-    Et turpis eget tempor. Nam sed lacus ut lorem feugiat tincidunt.
-    Mauris quis nisi placerat.
+  <Notification closeText="This is very long close button text.">
```

Comment:
I propose we have the first example with simple texts and change the closeText to "Close". Then the second example with error with error contents, so I would change the children to contain the word error also. Then a new, third example with long texts and the final one with simple texts and action buttons. How action button handle long texts is less irrelevant in this context.

Reply:
Fixed

Reply:
Currently having three examples, instead of the four suggested. I think the now existing three are good to go.